src/style.css

```diff
@@ -0,0 +1,21 @@
+.blue {
+  height: 100px;
+  background-color: blue;
+  width: 100%;
+}
+
+.green {
+  height: 100px;
+  background-color: green;
+  width: auto;
+}
+
+.red {
+  height: 100px;
+  background-color: red;
+  width: 100%;
```

Comment:
Block elements has 100% of the width of the parent container. Set width of 100% is redundant